packages/components/review-date/template.html

```diff
@@ -0,0 +1,6 @@
+<div class="nhsuk-c-review-date">
+  <p class="nhsuk-body-s">
+    Page last reviewed: 12/02/2016<br>
```

Comment:
@AdamChrimes - What do you think about surrounding the dates in a `<time>` element? For semantics.

Reply:
Resolved